### PR TITLE
Feat: Added Date and Currency Formatters

### PR DESCRIPTION
**Pull request by RobinCsl**
Summary: Date and Currency Formatters, extracted from mobile repository, rely on Intl library, which needs to be patched for it to work on Android.
This lead to a rewrite of the other patch in apps/mobile in Node, to have all the patches in the same file.

### CODE REVIEW

File: apps/mobile/scripts/configureApplication.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        '..',
+        '..',
+        'node_modules',
+        'get-yarn-workspaces',
+        file,
+      ),
+    );
+  });
+
+  // fs.unlinkSync(
```
Comment (FilipMessa) [inline apps/mobile/scripts/configureApplication.js]: Do we need this commented part?

File: packages/utils/localization/DateUtils.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+// @flow strict
+
+// test immutability
```
Comment (FilipMessa) [inline packages/utils/localization/DateUtils.js]: test immutability what does this mean? TODO?

File: apps/mobile/scripts/configureApplication.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
+// @flow strict
+
```
Comment (FilipMessa) [inline apps/mobile/scripts/configureApplication.js]: Can you write some comment or add it to readme why we need this script, please? thanks

File: packages/utils/localization/DateUtils.js (hunk 2 above)
Comment (RobinCsl) [reply packages/utils/localization/DateUtils.js]: This was extracted as is from `mobile` repository.

File: apps/mobile/scripts/configureApplication.js (hunk 3 above)
Comment (RobinCsl) [reply apps/mobile/scripts/configureApplication.js]: Sure, I'll add some comments.

File: packages/utils/localization/DateUtils.js (hunk 2 above)
Comment (RobinCsl) [reply packages/utils/localization/DateUtils.js]: Added @TODO

**Pull request merged**